core/elastic/config.go

```diff
@@ -41,12 +45,36 @@ type ElasticsearchConfig struct {
 	Endpoint     string `config:"endpoint"`
 	TemplateName string `config:"template_name"`
 	IndexPrefix  string `config:"index_prefix"`
+	IndexSuffix  string `config:"index_suffix"`
+	Engine       *struct {
+		EngineID  string `json:"engineId,omitempty" config:"engine_id"`
+		AccountID string `json:"accountId,omitempty" config:"account_id"`
+	} `json:"engine,omitempty" config:"engine"`
 	BasicAuth    *struct {
 		Username string `config:"username"`
 		Password string `config:"password"`
 	} `config:"basic_auth"`
 }
 
+
+func (c* ElasticsearchConfig) Check()  {
+	
+	// Fail fast if key things are missing
+	if c.IndexPrefix == "" {
```

Comment:
Should index suffix be checked as well?

Reply:
That's a good point. Suffix isn't technically required. I mean, we need it, but I probably shouldn't be representing our concerns in this code base. The same goes for the prefix. I think I am going just going to get rid of this check entirely.

Reply:
Decided to just add log.Warns instead of panics.